pkgs/tools/security/sudo/nspawn.nix

```diff
@@ -0,0 +1,18 @@
+{ sudo, fetchpatch, autoreconfHook, lib }:
+
+sudo.overrideAttrs ({ nativeBuildInputs ? [], configureFlags ? [], patches ? [], meta, ... }: {
+  patches = patches ++ [
+    (fetchpatch {
+      url = "https://github.com/sudo-project/sudo/commit/4b365300a70900f000886729431be93dbbc323b0.patch";
+      sha256 = "sha256-1CB2vBf5FIsDeTBllklcSdCHw59pKtixwGGALlsHScM=";
+    })
+  ];
```

Comment:
I just merged the sudo upgrade https://github.com/NixOS/nixpkgs/pull/153281 which means we can drop this patch now. Maybe also the autoreconfHook below? I am not sure.